src/main/cpp/t06_quad.cpp

```diff
@@ -21,5 +21,22 @@
 using namespace std;
 
 int t06_quad() {
-
-};
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'x' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4311533)